kind/README.md

```diff
@@ -0,0 +1,111 @@
+pod/kuma-injector-696484d998-9jprt condition met
+namespace/kuma-demo created
+deployment.apps/postgres-master created
+service/postgres created
+deployment.apps/redis-master created
+service/redis created
+service/backend created
+deployment.apps/kuma-demo-backend-v0 created
+deployment.apps/kuma-demo-backend-v1 created
+deployment.apps/kuma-demo-backend-v2 created
+service/frontend created
+deployment.apps/kuma-demo-app created
+pod/kuma-demo-app-68758d8d5d-6hmz5 condition met
+pod/kuma-demo-backend-v0-6fdb79ddfd-pg82b condition met
+pod/postgres-master-78d9c9c8c9-p7swm condition met
+pod/redis-master-657c58c859-9r57p condition met
+NAME                                 READY   STATUS    RESTARTS   AGE
+kuma-control-plane-965bf6fd4-tg8sx   1/1     Running   0          80s
+kuma-injector-696484d998-9jprt       1/1     Running   0          80s
+NAME                                    READY   STATUS    RESTARTS   AGE
+kuma-demo-app-68758d8d5d-6hmz5          2/2     Running   0          56s
+kuma-demo-backend-v0-6fdb79ddfd-pg82b   2/2     Running   0          56s
+postgres-master-78d9c9c8c9-p7swm        2/2     Running   0          56s
+redis-master-657c58c859-9r57p           2/2     Running   0          56s
+Error: Control Plane with name "kind" already exists. Use --overwrite to replace an existing one.
+MESH      NAME                                              TAGS                                                                                                                        STATUS   LAST CONNECTED AGO   LAST UPDATED AGO   TOTAL UPDATES   TOTAL ERRORS
+default   kuma-demo-app-68758d8d5d-6hmz5.kuma-demo          app=kuma-demo-frontend env=prod pod-template-hash=68758d8d5d protocol=http service=frontend.kuma-demo.svc:8080 version=v8   Online   3s                   2s                 4               0
+default   kuma-demo-backend-v0-6fdb79ddfd-pg82b.kuma-demo   app=kuma-demo-backend env=prod pod-template-hash=6fdb79ddfd protocol=http service=backend.kuma-demo.svc:3001 version=v0     Online   4s                   3s                 4               0
+default   postgres-master-78d9c9c8c9-p7swm.kuma-demo        app=postgres pod-template-hash=78d9c9c8c9 protocol=tcp service=postgres.kuma-demo.svc:5432                                  Online   4s                   2s                 4               0
+default   redis-master-657c58c859-9r57p.kuma-demo           app=redis pod-template-hash=657c58c859 protocol=tcp role=master service=redis.kuma-demo.svc:6379 tier=backend               Online   6s                   4s                 4               0
+Kuma GUI is available at http://localhost:5683/
+Kuma DEMO is available at http://localhost:8080/
+Type 'quit' to exit.
+```
+
+As the message states, the Demo site can be found at [http://localhost:8080](http://locahost:8080) and the Kuma GUI at [http://localhost:5683](http://locahost:5683). You can also use `kubectl` and `${HOME}/bin/kumactl` to explore the deployment and configuration.
```

Comment:
should we point them to the demo instructions on how to apply policies found in the Kubernetes readme?